python/custreamz/custreamz/kafka.py

```diff
@@ -173,7 +167,10 @@ def read_gdf(
         kafka_datasource.close(batch_timeout)
 
         if result is not None:
-            return cudf.DataFrame._from_table(result)
+            if isinstance(result, cudf.DataFrame):
+                return result
+            else:
+                return cudf.DataFrame._from_data(result)
```

Comment:
I'm confused, what changed in this PR to make this change necessary? Did this PR start before `DataFrame._from_table` was removed? It looks like it, but in the old code it was always calling `_from_table` so I'm curious when you would go into the `if` rather than the `else` here.

Reply:
Nothing in this PR. This has actually been broken ever since the _from_table changes occurred but was never noticed since this portion is rarely used. I choose to just make that quick fix in this PR.